controllers/humiocluster_controller_test.go

```diff
@@ -2169,6 +2179,66 @@ var _ = Describe("HumioCluster Controller", func() {
 			}, testTimeout, testInterval).Should(BeEquivalentTo(120))
 		})
 	})
+
+	Context("Humio Cluster install license", func() {
+		It("Should succesfully install a license", func() {
```

Comment:
Perhaps we should also include a test case where it tries installing an invalid license? Perhaps the HumioCluster.Status.State should even result in `ConfigError` if the referenced k8s secret does not exist, or if it does not contain the expected content?